__tests__/api.test.js

```diff
@@ -4,6 +4,7 @@ const db = require("../db/connection");
 const seed = require("../db/seeds/seed");
 const testData = require("../db/data/test-data/index");
 const endpointsJson = require("../endpoints.json");
+const jestSorted = require("jest-sorted");
```

Comment:
You don't need to require jest sorted in. You already have access to its matchers once installed 🙂